test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,31 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception{
+        // empty list
+        assertNotNull();
+
+        // 1 null items
+        assertAnyNull((Object) null);
+
+        // some null items
+        assertAnyNull(null, null);
+        assertAnyNull(null, 1,null);
+        assertAnyNull('a', null, "a", null);
```

Comment:
Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.